test/stdlib/queue-tests.delta

```diff
@@ -0,0 +1,38 @@
+// RUN: check_exit_status 0 %delta run -Werror %s test-support.delta
+
+
+func main() {
+    testPushPop();
+    testEmpty();
+}
```

Comment:
Can you add tests for:
- pushing after having popped
- calling isEmpty when the queue is not empty